### DISCUSSION TITLE
The version in GURU is newer than the version in AUR

### DISCUSSION THREAD

**Discussion by ceamac** [fastfetch-cli/fastfetch#400] [Show and tell]
This doesn't happen often :smile: 

![image](https://user-images.githubusercontent.com/50396932/213911217-4a9e2c45-e2e6-4988-a205-2eeedd839a20.png)

**Comment by ceamac** [comment]
And since we are here, my Arch box has uptime.
![image](https://user-images.githubusercontent.com/50396932/213911790-f54263d8-afe1-4e11-b5d3-7dbabbb4fde5.png)

**Reply by LinusDierheimer** [reply]
This is the first screenshot i see where the `(!)` behind `days` is displayed, i almost forgot i implemented that.

**Reply by luni3359** [reply]
What does that exclamation mark mean?

**Reply by LinusDierheimer** [reply]
Its a reference to htop: https://unix.stackexchange.com/questions/150121/what-does-the-mean-after-uptime-on-htop

**Reply by luni3359** [reply]
Thank you for explaining
